apps/fhir/bluebutton/views/search.py

```diff
@@ -139,7 +139,7 @@ def get_paging_links(base_url, start_index, page_size, count, replay_parameters)
             'url': base_url + '?' + urlencode(replay_parameters)
         })
 
-    if start_index - page_size > 0:
+    if start_index - page_size >= 0:
```

Comment:
looks good to me but we might want to have the `first` and `previous` relations both present and set to `0` if `start_index - page_size` goes negative. It'd make things easier for implementations using the api.

Reply:
Yeah, honestly, I don't know. I expect that no one will use these links and end up with off multiple numbers, so it likely doesn't matter, but you make a good point. I just wasn't sure what to do with these in those unusual cases.